src/tests/utils/test-utils.tsx

```diff
@@ -5,8 +5,8 @@ import { Provider } from 'react-redux';
 
 import { store } from '../../common/store.ts';
 
-function render(ui) {
-  function Wrapper({ children }) {
+function render(ui: React.ReactElement) {
+  function Wrapper({ children }: { children: React.ReactElement }) {
```

Comment:
This type is coming from: 
https://github.com/testing-library/react-testing-library/blob/f176285e4e92754751b708e1b1adf1f38edea6a8/types/index.d.ts#L135